package-lock.json

```diff
@@ -1718,7 +1718,6 @@
                 "jest-resolve": "^26.6.2",
                 "jest-util": "^26.6.2",
                 "jest-worker": "^26.6.2",
-                "node-notifier": "^8.0.0",
```

Comment:
Let's revert the changes to this file. Just curious, what's the output of `node -v` on your machine?

Reply:
node -v returns v15.14.0. What version are you using?

I ran node i to get all dependencies and that resulted in updating package-lock.json. Can you help me understand why that happened and why I should revert?

Reply:
I reverted the change I made to this file

Reply:
Mine is reporting `15.5.0`, so not much farther off yours. I'll update and see if the changes come through on my end as well.

As for the "why", generally we don't want to touch package-lock unless we're updating/adding dependencies to package.json. As for why yours updated the package-lock, specifically? I don't know. npm install can be a little finicky at times and I'm not an expert on why. It may be as simple as "your lock file is right and the committed version is wrong" or "new version, new lockfile rules". I'll take a look into it.